sycl/test-e2e/Matrix/elemwise_irreg_size_ops_bf16.cpp

```diff
@@ -14,31 +14,16 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+#include "common.hpp"
 #include <iostream>
 #include <sycl/sycl.hpp>
```

Comment:
Rebase to the latest sycl. E2E tests are not using sycl/sycl.hpp anymore

Reply:
No need to re-base. This change does not conflict with the changes committed in the past hour.

Reply:
I see you merged in latest changes. Thanks.